CHANGELOG.md

```diff
@@ -10,6 +10,7 @@
 - `[jest-fake-timers]` `getTimerCount` will not include cancelled immediates ([#8764](https://github.com/facebook/jest/pull/8764))
 - `[jest-leak-detector]` [**BREAKING**] Use `weak-napi` instead of `weak` package ([#8686](https://github.com/facebook/jest/pull/8686))
 - `[jest-snapshot]` Remove only the added newlines in multiline snapshots ([#8859](https://github.com/facebook/jest/pull/8859))
+- `[jest-circus]` Throw a proper error if a test / hooks is defined asynchronously ([#8096](https://github.com/facebook/jest/pull/8096))
```

Comment:
`[**BREAKING**]`? Also, these are alphabetically sorted (just for this new part, but still)

Reply:
Yeah might as well. Dunno if this was "officially" supported, but at least for hooks there was a test case. We'll also have to see if we have to first do this for tests only because for hooks it would be too breaking for FB, so let's wait what @scotthovestadt said about how to proceed